### DISCUSSION TITLE
[Feature]: How to Generate Curved Text ?

### DISCUSSION THREAD

**Discussion by niketaOptimumnbrew** [fabricjs/fabric.js#9754] [Q&A]
### CheckList

- [X] I agree to follow this project's Code of Conduct
- [X] I have read and followed the [Contributing Guide](https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md)
- [X] I have searched and referenced existing issues, feature requests and discussions
- [X] I am filing a **FEATURE** request.

### Description

I'm using a textbox to render text on a canvas. I want to generate or create curved text. Can anyone please tell me how to achieve that?
Please check the screenshot below; I want to achieve the same layout as shown in the screenshot.
[
![image](https://github.com/fabricjs/fabric.js/assets/126561906/d3d668b9-9ed2-4fac-b3fa-4ad9a0a85a7d)
](url)

### Current State

I'm using fabric.js version 5.3.0

**Comment by pramod-praella** [comment]
@niketaOptimumnbrew 
Do you have any updates on this.
